src/Ryujinx/Input/GTK3/GTK3Keyboard.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Ryujinx.Input.GTK3
 {
-    public class GTK3Keyboard : IKeyboard
+    public class Gtk3Keyboard : IKeyboard
```

Comment:
```suggestion
    public class GTK3Keyboard : IKeyboard
```

I'd prefer to keep `GTK`